sc-network/sctp_server/utils/parser.hpp

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
```

Comment:
Why we need separate config parser for sctp-server and sc-server. I think we can develop common config parser. Not only for servers